tests/ui/test_login.py

```diff
@@ -1,11 +1,25 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
+from seleniumlogin import force_login
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.django_db
+@pytest.mark.skipif('localhost' not in os.getenv('PYTEST_BASE_URL'),
+                    reason='No force login for dev, prod testing')
```

Comment:
We want to test the login functionality regardless of the target. I'm not sure what force login means, but it doesn't sound like what we want to do.